src/cli.ts

```diff
@@ -76,28 +76,56 @@ Provide it via the ${AUTOBLOCKS_API_KEY_NAME} environment variable, within a .en
 
 You can get your API key from ${AUTOBLOCKS_WEBAPP_BASE_URL}/settings/api-keys`;
 
+const commandNotFoundErrorMessage = `
+No command found. Provide a command following '--'. For example:
+
+npx autoblocks testing exec -- echo "Hello, world!
+`;
+
+const parseCommandFromArgv = (
+  argv: yargs.Arguments,
+): { command: string; commandArgs: string[] } => {
+  const unparsed = argv['--'];
```

Comment:
Just an understanding check - `--` is the argument name right?

Reply:
it's where yargs puts everything after `--`: https://github.com/autoblocksai/cli/blob/3417bddc9dcb4efc960da1396a07fe4653bb8b73/src/cli.ts#L142-L146